src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/config/HmppsAssessForEarlyReleaseApiExceptionHandler.kt

```diff
@@ -33,7 +33,7 @@ class HmppsAssessForEarlyReleaseApiExceptionHandler {
     ).also { log.info("Validation exception: {}", e.message) }
 
   @ExceptionHandler(HttpMessageNotReadableException::class)
-  fun handleHttpMessageNotReadableException(e: ValidationException): ResponseEntity<ErrorResponse> = ResponseEntity
+  fun handleHttpMessageNotReadableException(e: HttpMessageNotReadableException): ResponseEntity<ErrorResponse> = ResponseEntity
```

Comment:
This caused a 500 error when trying to process this type of error

Reply:
Would this causing other valid ValidationExceptions to be ignore?

Reply:
There is already an handler for ValidationException